Source/System/Drawing/ColorConverter.cs

```diff
@@ -114,7 +118,9 @@ public class ColorConverter : TypeConverter {
             }
             // Ok, how about a system color?
             //
+#if !NETSTANDARD20
             color = SystemColors[name];
+#endif
```

Comment:
It seems a bit inconsistent - `Colors` is changed to return an empty hashtable and is used as before, while `SystemColors` is not changed and instead the usage is commented out. I would probably handle `SystemColors` the same way as `Colors`.

Reply:
Actually Colors should not be commented, instead SystemColors should be. Pushed fix.